File: RecargaPlus/Controller/objects/Bonificacion.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Controller.objects
{
    public class Bonificacion
    {
        public int ID { get; set; }
        public double bonificacion { get; set; }
        public Compania Compania { get; set; }
        public Monto Monto { get; set; }
    }
}

File: Andtoid/app/src/main/java/Objects/Bonificacion.java
package Objects;

public class Bonificacion {
    private int id;
    private double bonificacion;
    public Compania compania;
    public Monto monto;

    public  Bonificacion(){
         this.compania = new Compania();
         this.monto = new Monto();
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public double getBonificacion() {
        return bonificacion;
    }

    public void setBonificacion(double bonificacion) {
        this.bonificacion = bonificacion;
    }


}
